beets/ui/commands.py

```diff
@@ -708,6 +736,15 @@ def choose_match(self, task):
                     _, _, candidates, rec = autotag.tag_album(
                         task.items, search_id=search_id
                     )
+            elif choice in extra_ops.keys():
+                # Allow extra ops to automatically set the post-choice.
+                # TODO: currently action.MANUAL and action.MANUAL_ID are not
+                # properly handled
+                post_choice = extra_ops[choice](self, task)
+                if post_choice in (importer.action.SKIP, importer.action.ASIS,
+                                   importer.action.TRACKS,
+                                   importer.action.ALBUMS):
+                    return post_choice
```

Comment:
Great, I'll just make sure to mention it on the documentation as it might not be evident for the plugin developers.